tests/helpers.py

```diff
@@ -86,14 +87,14 @@ def send_create_update_tx(name, ddo, flags, account):
 
     dt_contract = get_nft_contract(get_web3(), datatoken_address)
 
-    cap = web3.toWei(100000, "ether")
+    cap = web3.to_wei(100000, "ether")
     erc20_txn = dt_contract.functions.createERC20(
         1,
         ["ERC20DT1", "ERC20DT1Symbol"],
         [
-            web3.toChecksumAddress(account.address),
-            web3.toChecksumAddress(account.address),
-            web3.toChecksumAddress(account.address),
+            to_checksum_address(account.address),
```

Comment:
why do we need `eth_utils` here. web3 has support for converting to checksum address https://web3py.readthedocs.io/en/stable/web3.main.html#web3.Web3.to_checksum_address. maybe the import is not correct.

Reply:
it's much simpler like this. Web3 uses eth utils under the hood. As you can see, there is no dependency added, we are just using the one brought in by web3.py. Furthemore, previously we had two types of usages:
- Web3.toChecksumAddress (via class static method)
- web3.toChecksumAddress (using the object)

For the same result, it is better to remove the dependency on the object, maybe that way it can be elliminate passing it as an argument just for the sake of using a simple function like `to_checksum_address` or `is_address`. But if we eliminate the dependency on the object, I suggest also removing the class-based usages of to_checksum_address for uniformity.

I also think this will become much more apparent when brownie will upgrade web3.py and we'll have to do a similar PR in ocean.py. We have a lot of web3 object being passed around as an argument just for such trivial things.